keyboards/eggsworks/egg58/info.json

```diff
@@ -0,0 +1,171 @@
+{
+    "keyboard_name": "egg58",
+    "manufacturer": "eggsworks",
+    "url": "https://github.com/eggsworks/egg58",
+    "maintainer": "Travis Mick <travis@eggs.works>",
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0x3665",
+        "device_version": "2.1.1"
+    },
+    "features": {
+        "bootmagic": true,
```

Comment:
```suggestion
        "bootmagic": true,
        "extrakey": true,
        "mousekey": true,
```

Reply:
Good catch, I didn't realize that I lost extrakey when converting to JSON - it turns out some of the keys I had mapped weren't working. Thanks.